developer-docs-site/docs/basics/basics-accounts.md

```diff
@@ -15,7 +15,7 @@ An account may contain an arbitrary number of Move resources and Move modules.
 
 ## Initial Account Setup
 
-An Aptos account is referenced by an account address, which is a 16-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the last 16-bytes of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
+An Aptos account is referenced by an account address, which is a 32-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the last 16-bytes of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
```

Comment:
Actually, there's another reference to 16 bytes still left in here. Can you update that?

Reply:
Right, there are two contradictory sentences: "an account address, which is a 32-byte value" and "the account address is the last 16-bytes"

Reply:
just see the comments. will update soon